deploy/eck-stack/charts/eck-autoops/values.yaml

```diff
@@ -0,0 +1,59 @@
+---
+# Default values for eck-autoops.
+# This is a YAML-formatted file.
+
+# Overridable names of the AutoOps Agent Policy resource.
+# By default, this is the Release name set for the chart,
+# followed by 'eck-autoops'.
+#
+# nameOverride will override the name of the Chart with the name set here,
+# so nameOverride: quickstart, would convert to '{{ Release.name }}-quickstart'
+#
+# nameOverride: "quickstart"
+#
+# fullnameOverride will override both the release name, and the chart name,
+# and will name the AutoOps Agent Policy resource exactly as specified.
+#
+# fullnameOverride: "quickstart"
+
+# Version of AutoOps Agent Policy.
+#
+version: 9.3.0-SNAPSHOT
+
+# Labels that will be applied to AutoOps Agent Policy.
+#
+labels: {}
+
+# Annotations that will be applied to AutoOps Agent Policy.
+#
+annotations: {}
+
+# AutoOps configuration values.
+# These values are required and will be used to create a Kubernetes Secret.
+#
+# All of these can be obtained the Elastic Cloud Console (https://cloud.elastic.co/connect-cluster-services-portal)
+# by selecting the ECK connect wizard and following the instructions.
+#
+# This is the Cloud Connected Mode API key and is required. This is used to register the Agent with Elastic Cloud AutoOps.
+ccmApiKey: ""
+# This is the URL of the AutoOps OTel endpoint and is required. This is used to send Agent data to the AutoOps platform and is region specific.
+# Example: "https://otel-auto-ops.us-east-2.aws.svc.elastic.cloud"
+autoOpsOTelURL: ""
+# This token is required and is how AutoOps knows how to connect Agent data to the relevant Elastic Cloud organization
+autoOpsToken: ""
+# URL of the Cloud Connected Mode API.
+# This is not a required field.
+cloudConnectedModeAPIURL: ""
```

Comment:
since in the autoopsagentpolicy this is a ConfigSource, drills down to a secretRef, I believe it makes sense to allow the users to specify directly an external-managed secret name here